_episodes/04-function-style.md

```diff
@@ -356,9 +408,13 @@ def calculate_distance(rA, rB):
 
 ## Docstrings
 We've now added a multi-line comment (called a `docstring`, short for "documentation string"), to the beginning of our function. Docstrings **are the first statement after a function or module definition** and are opened and closed with three quotes.
-
 The docstring should explain what the function or module does (and not how it is done).
 
+[PEP257] provides very basic guidelines for docstrings.
```

Comment:
Provide link here `[PEP257](https://www.python.org/dev/peps/pep-0257/)` or put link in section at bottom of file under `{% include links.md %}`

Reply:
Is the `{% include links.md %}` necessary? Several files are missing that.

Reply:
Is this resolved?

Reply:
Hi Eric, I remember typing a response to this, but I don't see it here so I either dreamed about it, or responded and forgot to send. :)

`{% include links.md %}` refers to the file `_includes/links.md`. I believe this line inserts all the links from `links.md` similar to the way the lines below the include statement are included. For example, in lesson 2, we have `[GitHub]: https://github.com`. This makes it so you can jut put `[GitHub]` each time you want to link to github rather than `[GitHub](https://github.com)`. If it were in `links.md` this short cut would be accessible to any file that includes `links.md` without needing to be defined.

`links.md` is a file that lets you define commonly used links and insert them into all lessons. It may not be necessary for many of the lessons, but I believe it was included in the template by default and has never been removed. It may be a good idea to start using the file more wisely, however.